react/ecmascript/02 - promises/index.js

```diff
@@ -1,20 +1,12 @@
 export function delay(TIME) {
-  const start = Date.now();
   const reason = new Error('This time is too much !');
   return new Promise((resolve, reject) => {
     if (TIME > 500) {
       reject(reason);
     } else {
-      setInterval(() => {
```

Comment:
You still need the `setInterval`. You were supposed to delete the `if (delayedTime <= TIME + 100 || delayedTime >= TIME - 100) {` only